fastserde/avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastGenericDeserializerGenerator.java

```diff
@@ -6,7 +6,7 @@
 
 public final class FastGenericDeserializerGenerator<T> extends FastDeserializerGenerator<T> {
 
-  FastGenericDeserializerGenerator(Schema writer, Schema reader, File destination, ClassLoader classLoader,
+  public FastGenericDeserializerGenerator(Schema writer, Schema reader, File destination, ClassLoader classLoader,
```

Comment:
I guess that they left the visibility of the constructors limited due to the testing approach that I mentioned.

Reply:
Due to Gradle config I had to create sub-package `com.linkedin.avro.fastserde.logical.types` which doesn't have access to package-private c-tor of `FastGenericDeserializerGenerator`.

I may create helper factory in src/test/java in `com.linkedin.avro.fastserde` package which has access to package-private c-tors `FastGenericDeserializerGenerator ` and `FastGenericSerializerGenerator`.

On the other hand `FastSpecificSerializerGenerator` and `FastSpecificDeserializerGenerator` have public c-tors which is the reason why I made `FastGenericDeserializerGenerator` c-tor public.

Please share your thoughts.